services/velero/5.2.2/helmrelease/helmrelease.yaml

```diff
@@ -3,6 +3,8 @@ kind: HelmRelease
 metadata:
   name: velero
   namespace: ${releaseNamespace}
+  annotations:
+    velero.kommander.d2iq.io/no-hooks-chart: "true"
```

Comment:
clever!

Reply:
Initially I intended to watch for the proper chart version in the HelmRelease, but realized that this would be very prone to breakage in the face of patch releases. We would have either needed one more location where the version needs to be bumped, or in a somewhat non-trivial propagation of said version.